### PR TITLE
Catalog component

### PR DESCRIPTION
**Pull request by 95kach**
https://github.com/capsulajs/capsulahub-ui/issues/11

### CODE REVIEW

**Review by VictoriaBordian — APPROVED** [review]
QA passed

File: src/components/catalog/catalog.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import List from './list';
+
+const Container = styled.div`
```
Comment (MikeSquall) [inline src/components/catalog/catalog.js:6]: Styled component shouldn't be in a separate file with all styled components ?

File: src/components/catalog/list.js [hunk 2]
```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
```
Comment (MikeSquall) [inline src/components/catalog/list.js:3]: Styled component shouldn't be in a separate file with all styled components ?

File: src/docs/catalog.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Catalog } from '../components';
+
+const onSelect = (tab) => console.log('Select: ', tab);
+const menu = [
+  {
+    id: 0,
+    name: 'Enviroment',
```
Comment (MikeSquall) [inline src/docs/catalog.js]:
```suggestion
    name: 'Environment',
```

File: src/components/catalog/catalog.js (hunk 1 above)
Comment (95kach) [reply src/components/catalog/catalog.js:6]: it's a choice of taste, I like this minimalistic approach

File: src/components/catalog/list.js (hunk 2 above)
Comment (95kach) [reply src/components/catalog/list.js:3]: No.. I don't see any profit in it.

File: src/components/catalog/catalog.js (hunk 1 above)
Comment (95kach) [reply src/components/catalog/catalog.js:6]: I don't want to always keep opened 100500 files in my editor )

**Review by MikeSquall — APPROVED** [review]

**Pull request merged**